server/fishtest/templates/signup.mak

```diff
@@ -8,8 +8,8 @@
 
 <div>
   <p></p>
-  <p>Thank you! Sign up to fishtest to contribute either CPU time and patches to test.</p>
-  <p>Once a new user account is created, an approver needs to manually activate it. This is usually quick, but sometimes takes a few hours.</p>
+  <p>Thank you! Sign up to fishtest to contribute either CPU time or code patches to be tested.</p>
+  <p>An approver will review your new account to avoid spam. This is usually quick, but sometimes takes a few hours.</p>
```

Comment:
I would prefer:

`A person will now manually approve your new account, to avoid spam.`

Reply:
Dropped "now" here.